lib/marked.js

```diff
@@ -1342,22 +1346,23 @@ marked.setOptions = function(opt) {
 };
 
 marked.defaults = {
-  gfm: true,
-  tables: true,
+  baseUrl: null,
   breaks: false,
+  gfm: true,
+  headerPrefix: '',
+  headerIds: true,
```

Comment:
'I' comes before 'P'

Reply:
Good catch...funny enough, I got it right in the table. :)